Makefile

```diff
@@ -37,6 +37,9 @@ docker-run:
 # Go
 #############################
 
+.PHONY: generate
+generate: proto-all mockgen
```

Comment:
The rule name is too abstract. It can be anything.
Please specify what it generates.

Reply:
That's what I expect this command to run all generate commands.
Do you have any idea?

Reply:
If you expect to generate all the commands, how about `generate-all` or `generate-all-commands`?

Reply:
`generate-all` sounds look to me 👍